src/ontology.rs

```diff
@@ -745,6 +758,52 @@ impl Ontology {
         self.omim_diseases.values()
     }
 
+    /// Returns a vector contains diseases whose names contains the provided
```

Comment:
I think you forgot to also update the documentation to clarify that the function is now returning iterator and not a vector